indico/modules/events/registration/clone.py

```diff
@@ -22,47 +22,85 @@
 from indico.modules.events.registration.models.forms import RegistrationForm
 from indico.modules.events.registration.models.form_fields import RegistrationFormFieldData
 from indico.modules.events.registration.models.items import RegistrationFormItem, RegistrationFormSection
+from indico.modules.events.registration.models.registrations import Registration, RegistrationData
 from indico.util.i18n import _
 
 from MaKaC.conference import EventCloner
 
 
 class RegistrationFormCloner(EventCloner):
+    def __init__(self, *args, **kwargs):
+        super(RegistrationFormCloner, self).__init__(*args, **kwargs)
+        self.field_data_dict = {}
+        self._clone_with_registrations = False
+
     def get_options(self):
         if not is_feature_enabled(self.event, 'registration'):
             return {}
         enabled = bool(self._find_registration_forms().count())
-        return {'form': (_('Registration forms'), enabled, False)}
+        return {
+            'form': (_('Registration forms'), enabled, False),
+            'registrations': (_('Registrations'), enabled, False)
```

Comment:
Should be selected by default if `event.getType() != 'conference'`

What about showing its title as 'Participants' (and hiding the form-only option) if there's only a single registration form with is_participation == true? (opinions @pferreir  @OmeGak?)